src/config/metrics.ts

```diff
@@ -0,0 +1,68 @@
+import { diag, DiagConsoleLogger, DiagLogLevel } from '@opentelemetry/api';
```

Comment:
Lets also disable this and maybe add a comment to enabled it if needed for debugging.